validation/src/test/scala/hmda/validation/rules/lar/validity/V230Spec.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ LoanApplicationRegister }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V230Spec extends LarEditCheckSpec with BadValueUtils {
+  property("Occupancy must be 1, 2, 3") {
+    forAll(larGen) { lar =>
+      lar.mustPass
+    }
+  }
+
+  val badLienStatusGen: Gen[Int] = intOutsideRange(1, 3)
+
+  property("Occupancy other than 1, 2, or 3 is invalid") {
+    forAll(larGen, badLienStatusGen) { (lar: LoanApplicationRegister, x: Int) =>
+      val invalidLoan = lar.loan.copy(occupancy = x)
+      val invalidLar = lar.copy(loan = invalidLoan)
+      invalidLar.mustFail
+    }
+  }
+
+  override def check: EditCheck[LoanApplicationRegister] = V230
+}
```

Comment:
This 🚫 ↩️  symbol happens when your text editor doesn't add a newline character at the end of the file. I'm not sure why it added one correctly in `V230.scala`, but not in `V230Spec.scala`. Any idea?